TypeCobol/Compiler/Directives/TypeCobolOptions.cs

```diff
@@ -13,6 +13,14 @@ public class TypeCobolOptions : IBMCompilerOptions, ITypeCobolCheckOptions
         // insert options specific to TypeCobol here ...
 #if EUROINFO_RULES
         public bool AutoRemarksEnable { get; set; }
+        /// <summary>
+        /// Path to generrate collected used copy names
+        /// </summary>
+        public string ReportUsedCopyNamesPath { get; private set; }
+        /// <summary>
+        /// Collected CopyNames if -cpyr report is selected.
+        /// </summary>
+        public System.Collections.Generic.Dictionary<string, System.Collections.Generic.HashSet<string>> CollectedCopyNames;
```

Comment:
I don' think this should be held in the options class. Look at `CLI` class, I think we already have everything we need, we already collect used copies, the format is different here because we also want to list the variations for each.

Currently we store used copies in a `HashSet<string>` in `CLI`, so let's switch to a `Dictionary<string, HashSet<string>>` like you suggested and depending on the option `-exc` or `-cpyr` we change the format of the output file. The method `WriteCopiesFile` would have to be adapted.

Reply:
Yes it's true but the purpose here is to be able to collect information out of CLI class, using only TypeCobol.dll

Reply:
Same remark as above, we should be able to rely only on `CopyTextNamesVariations` without having to create a new data storage. Besides `TypeCobolOptions` should not be concerned with data collection.

Reply:
Ok I understand

Reply:
done in 
016fda9